carbonmark-api/src/models/User.model.ts

```diff
@@ -17,6 +17,7 @@ export const UserModel = Type.Object({
   listings: Type.Array(ListingModel),
   activities: Type.Array(ActivityModel),
   assets: Type.Array(AssetModel),
+  nonce: Type.Optional(Type.Number()),
```

Comment:
I'm probably misunderstanding something. Isn't the nonce only supposed to be valid for a single auth attempt to prevent MITM attacks?

Reply:
Correct, it's included here because the client needs to know the correct nonce to sign. It's valid for a single successful PUT or POST and can never be reused (because it increases monotonically).

Previously we used random nonces which theoretically could reappear and thus be reused (to repeat an edit, so not really a problem but still technically worse)